docs/quickstarts/react.mdx

```diff
@@ -3,34 +3,85 @@ title: Use Clerk with React
 description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your React application.
 ---
 
-import {Steps} from "nextra-theme-docs";
-import { Callout } from "nextra-theme-docs";
-import { YouTube } from "@/components/Youtube";
-import { Images } from "@/components/Images";
-import {Cards} from "@/components/Cards";
-import { CodeBlockTabs } from "@/components/CodeBlockTabs";
+<TutorialHero 
+  quickstart="react"
+  exampleRepo={[ 
+    {
+      title: "React Quickstart Repo",
+      link: "https://github.com/clerk/clerk-react-quickstart"
+      
+    }
+  ]}
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk"
+    },
+    {
+      title: "Create a React application using Vite",
+      link: "https://vitejs.dev/guide/#scaffolding-your-first-vite-project"
+    }
+  ]}
+>
+- Use Clerk with React
+- Install `@clerk/clerk-react`
+- Set environment keys
+- Wrap your app in `<ClerkProvider />`
+- Embed the `<UserButton />` and `<SignInButton />`
+</TutorialHero>
+
+
 
 # Use Clerk with React
 
 Learn how to use Clerk to quickly and easily add secure authentication and user management to your React application.
 
 <Steps>
 
+### Install React using Vite
+
+We don't recommend using `create-react-app` for scaffoling your new React application. Instead we recommend using Vite to setup your application. 
+
+<CodeBlockTabs type="installer" options={["npm", "pnpm", "yarn"]}>
```

Comment:
So everywhere else in the docs, we have it npm -> yarn -> pnpm
Is there a reason the order changed? Was it discussed? We should keep it the same across the docs for consistency, so whatever we decide here needs to be reflected everywhere else and vice versa.

Reply:
I just added it that way. I'm not strongly attached to this.